src/main/java/webserver/http/response/HttpResponse.java

```diff
@@ -0,0 +1,84 @@
+package webserver.http.response;
+
+import webserver.http.Header;
+import webserver.http.request.requestline.Protocol;
+import webserver.http.request.requestline.ProtocolType;
+import webserver.http.request.requestline.Version;
+import webserver.http.response.statusline.StatusCode;
+import webserver.http.response.statusline.StatusLine;
+
+import java.util.Map;
+
+public class HttpResponse {
+
+    private StatusLine statusLine;
+    private Header header;
+    private byte[] body;
+
+    public HttpResponse(StatusLine statusLine, Header header, byte[] body) {
+        validate(statusLine, header);
+        this.statusLine = statusLine;
+        this.header = header.add("Content-Length", String.valueOf(body.length));
+        this.body = body;
+    }
+
+    public static HttpResponse of(StatusLine statusLine, Header header, byte[] body) {
+        return new HttpResponse(statusLine, header, body);
+    }
+
+    public static HttpResponse of(StatusLine statusLine, Header header) {
+        return new HttpResponse(statusLine, header, new byte[0]);
+    }
+
+    public static HttpResponse ok(Header header, byte[] body) {
+        return new HttpResponse(StatusLine.of(new Protocol(ProtocolType.HTTP, Version.ONE_ONE), StatusCode.OK), header, body);
```

Comment:
한 줄에서 너무 많은 처리를 하고 있네요.
정적 팩터리 메서드를 차라리 이렇게 써보면 어떨까요?

```suggestion
        return new HttpResponse(StatusLine.ofHttp11(), header, body);
```